crates/pathfinder/src/rpc/v02/method/get_proof.rs

```diff
@@ -0,0 +1,215 @@
+use anyhow::{anyhow, Context};
```

Comment:
Please move this module to a new directory `rpc/pathfinder/methods`. Essentially you need to mimic the `rpc/<some_api>/method` structure as in `v02`.

Reply:
6cdf7c3921b7ba5886b72789496c0386caa51591

Reply:
btw used `methods` but `v02` uses `method` . I think `methods` makes more sense

Reply:
Honestly I didn't pay attention to if it should be singular or plural, no problem. Let's just make it follow one convention in the [next PR to come, which will expose the method](https://github.com/eqlabs/pathfinder/pull/726#discussion_r1043235972).